resource/writers/match_writers.hpp

```diff
@@ -74,6 +76,24 @@ class sim_match_writers_t : public match_writers_t
     std::stringstream m_out;
 };
 
+/*! K8S match writers class for a matched resource set
+ */
+class k8s_match_writers_t : public match_writers_t
+{
+public:
+    virtual ~k8s_match_writers_t () {}
+    virtual int emit (std::stringstream &out);
+    virtual bool empty ();
+     virtual int emit_json (json_t **o, json_t **aux = nullptr);
+    virtual int emit_vtx (const std::string &prefix,
+                           const f_resource_graph_t &g, const vtx_t &u,
+                           unsigned int needs, bool exclusive);
+    virtual std::string get_node ();
+private:
+    std::stringstream m_out;
+    std::string nodename;
```

Comment:
Fluxion designates member variables with `m_`:
```suggestion
    std::string m_nodename;
```